### DISCUSSION TITLE
Notifications not arriving (SMTP or Gotify)

### DISCUSSION THREAD

**Discussion by likwid101** [seerr-team/seerr#396] [General]
I have set up email notifications and gotify and every time I test it nothing arrives in my inbox or on gotify. I mess with the email settings and it will throw an error in the logs but then set it back properly and no error & no message arrives for testing. I've tested email notifications from other services and it is working. I'm so confused why notifications are not making their way out of the program or where to even look where the problem is. Any help would be appreciated.

I"m running a docker image in unraid.

Thanks!

**Comment by fallenbagel** [comment]
Could you come to discord as this is a troubleshooting matter and is easier handled than discussion here? I'll post the solution here afterwards
